validator/tools/validate_rules.js

```diff
@@ -40,18 +40,7 @@ const RULES = [
         apply: def => {
             const notAllowed = [
                 // Types with not exposed functionality.
-                "INFRA-AZUREVIRTUALNETWORKSPUBLICIPADDRESS",
-                "INFRA-CONTAINER",
-                "INFRA-COUCHBASENODE",
-                "INFRA-F5NODE",
-                "INFRA-F5POOL",
-                "INFRA-F5POOLMEMEBER",
-                "INFRA-F5VIRTUALSERVER",
-                "INFRA-F5VIRTUALSERVER",
-                "INFRA-HOST",
                 "INFRA-KUBERNETESCLUSTER",
-                "INFRA-RABBITMQNODE",
```

Comment:
yay! why we can't expose K8s?

Reply:
Because it has the unsupported `ENTITY_NAME` 😞

Reply:
Oh true... We can keep that for a follow up I think. Because we will need the same in GM